kotlinx-coroutines-test/common/src/TestBuilders.kt

```diff
@@ -580,6 +580,6 @@ internal expect fun dumpCoroutines()
 public fun TestScope.runTestLegacy(
     dispatchTimeoutMs: Long,
     testBody: suspend TestScope.() -> Unit,
-    unused1: Int,
+    marker: Int,
     unused2: Any?,
```

Comment:
Maybe rename the parameters to `mask: Int, marker: Any?`.

Reply:
Ah, I'm too slow. Well it shouldn't really matter.

Reply:
I disagree. `unused2` gives this function a tongue-in-a-cheek charm, which it deserves.